cli-output/Cargo.toml

```
@@ -26,6 +26,9 @@ solana-sdk = { path = "../sdk", version = "=1.7.0" }
 solana-stake-program = { path = "../programs/stake", version = "=1.7.0" }
 solana-transaction-status = { path = "../transaction-status", version = "=1.7.0" }
 solana-vote-program = { path = "../programs/vote", version = "=1.7.0" }
+spl-memo-v1 = { package = "spl-memo", version = "=1.0.2" }
+spl-memo-v2 = { package = "spl-memo", version = "=2.0.1", features = ["no-entrypoint"] }
+spl-memo-v3 = { package = "spl-memo", version = "=3.0.0", features = ["no-entrypoint"] }
```

```suggestion
spl-memo = { version = "=3.0.1", features = ["no-entrypoint"] }
```
We just published this patch, so you can get both program IDs from the same dependency, see usage below.
This should resolve the downstream-projects CI failure.